src/v2/view-builder/views/TerminalView.js

```diff
@@ -1,14 +1,32 @@
-import { createCallout } from 'okta';
+import { createCallout, loc } from 'okta';
 import BaseView from '../internals/BaseView';
 import BaseForm from '../internals/BaseForm';
+import BaseFooter from '../internals/BaseFooter';
+import {getBackToSignInLink} from '../utils/LinksUtil';
 
 const Body = BaseForm.extend({
   noButtonBar: true,
+  isTerminalForExpiredLink: false,
```

Comment:
TerminalView may contain different other links as well in future.. So instead of having a specific boolean here we can change this into an object that can contain any type of links.
`terminalLinks : {}

and later initialize accordingly
this.terminalLinks['expiredLinks'] = <some statement>

and use it
if(this.terminalLInks['expireLinks'])
`

Reply:
we can add those later once have it, currently added single method that checks the state

Reply:
Let's fix it in this PR.